tsqa/conf_plugin.py

```diff
@@ -0,0 +1,28 @@
+import logging
+import os
+
+from nose.plugins import Plugin
+
+log = logging.getLogger('nose.plugins.ConfPlugin')
+
+class ConfPlugin(Plugin):
+
+    name = 'conf-plugin'
+    def options(self, parser, env=os.environ):
+        """Register commandline options.
+        """
+        Plugin.options(self, parser, env)
+        parser.add_option('--keep-tmp', action='store_true', dest='keep_tmp', default=True,
```

Comment:
I wouldn't call it "keep-tmp" since where the environments are stored are configurable (read: not always in tmp). Probably want to rename this to "keep-env" or something similar